prod/westeurope/internal/api/apim/api_management_api_backoffice/swagger.json.tmpl

```diff
@@ -236,13 +293,100 @@
       }
     }
   },
-  "consumes": [
-    "application/json"
-  ],
-  "produces": [
-    "application/json"
-  ],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
   "definitions": {
+    "Wallet": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PublicWalletItem"
+          }
+        }
+      },
+      "required": ["data"]
+    },
+    "PublicCreditCard": {
+      "type": "object",
+      "properties": {
+        "hpan": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "enum": ["Bancomat", "Card"]
+        },
+        "masked_pan": {
+          "type": "string"
+        },
+        "holder": {
+          "type": "string"
+        },
+        "expireMonth": {
+          "type": "string"
+        },
+        "expireYear": {
+          "type": "string"
+        },
+        "brand": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "hpan",
+        "type",
+        "masked_pan",
+        "holder",
+        "expireMonth",
+        "expireYear",
+        "brand"
+      ]
+    },
+    "PublicBPay": {
+      "type": "object",
+      "properties": {
+        "hpan": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "enum": ["BPay"]
+        },
+        "back_name": {
+          "type": "string"
+        }
+      },
+      "required": ["hpan", "type", "back_name"]
```

Comment:
We have a type here. Maybe a fix is needed even on `fn-backoffice` specs file
```suggestion
        "bank_name": {
          "type": "string"
        }
      },
      "required": ["hpan", "type", "bank_name"]
```

Reply:
`io-fn-backoffice` spec fixed in pagopa/io-functions-backoffice#56

Reply:
thanks @BurnedMarshal, updated